openssl/run_envoy_docker.sh

```diff
@@ -1,43 +1,11 @@
 #!/bin/bash
 
-set -x
```

Comment:
I would rather keep `x`, is it useful for debugging.

Reply:
I know, and sometimes two line long log is very useful thing to have! :).
But I am ok with removing it too. I can still add it locally ;).